src/pages/bug-bounty/index.tsx

```diff
@@ -16,7 +16,7 @@ const meta_attributes = {
 
 const Security = () => {
     return (
-        <Layout type="security" margin_top={10}>
```

Comment:
this seems to be needed, or this removal is not related to your changes, can you please confirm?

Reply:
We added `margin-top` to the layout component whenever the type is `"security"`. Hence `margin_top` is not needed anymore.